source/renderer/app/components/staking/redeem-itn-rewards/Step1ConfigurationDialog.js

```diff
@@ -0,0 +1,391 @@
+
+    const buttonClasses = classnames([
+      'primary',
+      isSubmitting ? styles.isSubmitting : null,
+    ]);
+
+    const actions = {
+      direction: 'column',
+      items: [
+        {
+          className: buttonClasses,
+          disabled: !this.canSubmit,
+          primary: true,
+          label: intl.formatMessage(messages.continueButtonLabel),
+          onClick: this.submit,
+        },
+        {
+          onClick: (event: MouseEvent) =>
+            openExternalLink(
+              intl.formatMessage(messages.learnMoreLinkUrl, event)
+            ),
+          label: intl.formatMessage(messages.learnMoreLinkLabel),
+          isLink: true,
+        },
+      ],
+    };
+
+    const itnLink = (
+      <Link
+        className={styles.itnLink}
+        onClick={(event: MouseEvent) =>
+          openExternalLink(
+            intl.formatMessage(messages.descriptionItnLinkUrl, event)
+          )
+        }
+        label={intl.formatMessage(messages.descriptionItnLinkLabel)}
+        skin={LinkSkin}
+      />
+    );
+
+    return (
+      <>
+        <DialogCloseButton
+          className={redeemDialogOverride.closeButton}
+          onClose={onClose}
+        />
+        <Dialog
+          title={intl.formatMessage(messages.title)}
+          actions={actions}
+          onContinue={onContinue}
+          onClose={onClose}
+          customThemeOverrides={redeemDialogOverride}
+          closeOnOverlayClick={false}
+        >
+          <div className={styles.component}>
+            <p className={styles.description}>
+              <FormattedMessage
+                {...messages.description1}
+                values={{
+                  itnLink,
+                }}
+              />
+              <FormattedHTMLMessage {...messages.description2} />
+            </p>
+            <Autocomplete
+              {...recoveryPhraseField.bind()}
+              ref={autocomplete => {
+                this.recoveryPhraseAutocomplete = autocomplete;
+              }}
+              options={suggestedMnemonics}
+              maxSelections={WALLET_RECOVERY_PHRASE_WORD_COUNT}
+              error={recoveryPhraseField.error}
+              maxVisibleOptions={5}
+              noResultsMessage={intl.formatMessage(messages.noResults)}
+              className={styles.recoveryPhrase}
+              skin={AutocompleteSkin}
+              optionHeight={50}
+            />
+            <WalletsDropdown
+              className={styles.walletsDropdown}
+              {...walletsDropdownField.bind()}
+              numberOfStakePools={4}
+              wallets={wallets}
+              onChange={onSelectWallet}
+              placeholder={intl.formatMessage(
+                messages.selectWalletInputPlaceholder
+              )}
+              value={walletId}
+              getStakePoolById={() => {}}
+              error={this.walletsDropdownError}
+              errorPosition="bottom"
+            />
+            <hr />
```

Comment:
@daniloprates maybe would be better to use css and style margins than using `<hr>`?